### PR TITLE
Fix code style in Client-Side Storage recipe

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: src/v2/cookbook/client-side-storage.md [hunk 1]
```diff
@@ -140,13 +146,16 @@ const app = new Vue({
   methods: {
     addCat() {
       // ensure they actually typed something
-      if(!this.newCat) return;
+      if (!this.newCat) {
```
Comment (sdras) [inline src/v2/cookbook/client-side-storage.md]: I'm not sure this one needs to be fixed this way, maybe @chrisvfritz can sound off on it

**Comment by sdras** [comment]
Thank you for doing this! It was on my list to take care of. I just need a response from Chris on that one part and I can merge it in :)

**Comment by lex111** [comment]
As they say in many manuals on clean code and just good books, it is better to always put curly braces, even if only one expression :smiley:  Readability and easier to maintain - priceless. But the last word, of course, is yours.

**Comment by sdras** [comment]
You know, I don't care enough about it to stall this PR, it might be nicer to read for newcomers and we can always update if necessary.

**Pull request merged**